Error while checking build/O2/fullCI_slc9 for cb336fb364969c04ada134aafa81dda6bf992041 at 2025-09-09 16:06:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:272:41: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:278:41: error: reference to 'kEtaPrime' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:306:41: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:318:41: error: reference to 'kEtaPrime' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:340:14: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:343:14: error: reference to 'kEtaPrime' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:375:35: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:381:35: error: reference to 'kEtaPrime' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/Tasks/taskSingleElectron.cxx:338:50: error: could not convert '{{kPi0, Pi0}, {<expression error>, Eta}, {kOmega, Omega}, {o2::constants::physics::kPhi, Phi}, {<expression error>, EtaPrime}, {kRho770_0, Rho0}, {kKPlus, Ke3}, {kK0Long, K0l}}' from '<brace-enclosed initializer list>' to 'const std::map<int, SourceType>'
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx:572:45: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx:582:51: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx:590:54: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx:597:54: error: reference to 'kEta' is ambiguous
/sw/SOURCES/O2Physics/14662-slc9_x86-64/0/PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx:604:57: error: reference to 'kEta' is ambiguous
ninja: build stopped: subcommand failed.


## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
grep: error-log.txt: binary file matches
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/345521b3820f123fbc3954339692730de60e22a8/slc9_x86-64/o2checkcode/1.0-local916/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14662/cb336fb364969c04ada134aafa81dda6bf992041/build_O2_fullCI_slc9/pretty.html).

--- a/Common/Constants/include/CommonConstants/PhysicsConstants.h
+++ b/Common/Constants/include/CommonConstants/PhysicsConstants.h
@@ -31,6 +31,9 @@ namespace o2::constants::physics
 /// \note Follow kCamelCase naming convention
 /// \link https://root.cern/doc/master/TPDGCode_8h.html
 enum Pdg {
+  kEta = 221,
+  kOmega = 223,
+  kEtaPrime = 331,
   kB0 = 511,
   kB0Bar = -511,
   kBPlus = 521,
@@ -93,6 +96,9 @@ enum Pdg {
 };
 
 /// \brief Declarations of masses for additional particles
+constexpr double MassEta = 0.547862;
+constexpr double MassOmega = 0.78266;
+constexpr double MassEtaPrime = 0.95778;
 constexpr double MassB0 = 5.27966;
 constexpr double MassB0Bar = 5.27966;
 constexpr double MassBPlus = 5.27934;

--- a/Common/Constants/include/CommonConstants/make_pdg_header.py
+++ b/Common/Constants/include/CommonConstants/make_pdg_header.py
@@ -86,6 +86,9 @@ class PdgROOT(Enum):
 
 # Enum of additional particles
 class Pdg(Enum):
+    kEta = 221
+    kOmega = 223
+    kEtaPrime = 331
     kB0 = 511
     kB0Bar = -511
     kBPlus = 521

